upgrade/Stage_0_Prerequisites.md

```diff
@@ -46,6 +46,13 @@ after a break, always be sure that a typescript is running before proceeding.
 
 1. (`ncn-m001#`) Install the latest `docs-csm` RPM.
 
+   - This step should ONLY be performed if an http proxy is required to access a public endpoint on the internet for the purpose of downloading artifacts. CSM does NOT support the use of proxy servers for anything other than downloading artifacts from external endpoints. The http proxy variables must NOT be exported and must be unset after the desired artifacts are downloaded. Failure to unset the http proxy variables after downloading artifacts will cause many failures in subsuquent install steps.
```

Comment:
This is part of the upgrade procedure so may want to say "upgrade steps" instead of "install steps".